setup.cfg

```diff
@@ -7,4 +7,5 @@ exclude = mypy/codec/*,mypy/test/data/lib-stub/*,mypy/test/data/fixtures/*
 #   F401: unused identifiers (useless, as it doesn't see inside # type: comments)
 #   W601: has_key() deprecated (false positives)
 #   E701: multiple statements on one line (colon) (we use this for classes with empty body)
-ignore = E251,E128,F401,W601,E701
+#   F403: 'from foo import *' used; unable to detect undefined names (mypy checks this better)
```

Comment:
PEP 8 does not forbid glob imports, merely discourage them. And it specifically talks about "Modules that are designed for use via `from M import *`" - and `mypy.myunit.assertions` is such a unit.

If you think a different one of the 3 alternatives is more feasible, please comment in the other place where I listed them.